test/per_file_coverage.sh

```diff
@@ -23,7 +23,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/thread:0.0" # Death tests don't report LCOV
 "source/common/watchdog:58.6" # Death tests don't report LCOV
 "source/exe:90.3"
-"source/extensions/clusters/common:91.5" # This can be increased again once `#24903` lands
+"source/extensions/clusters/common:90.0" # This can be increased again once `#24903` lands
```

Comment:
I know this is a draft but please make sure it doesn't lower coverage - add more tests as needed.

Reply:
@alyssawilk 
Sure, this coverage change is just to verify whether the other changes can pass CI.
BTW,  we are going to put this draft PR on hold.  @jmarantz is trying to fix the issue with another approach: https://github.com/envoyproxy/envoy/pull/32919.  Could you please take a look that PR?

Reply:
Will not be reducing coverage in that PR either. Will try to increase it.

Reply:
If there is a need to revive this PR, we will certainly add some unit tests for logical host.